apps/web/app/(use-page-wrapper)/settings/(settings-layout)/layout.tsx

```diff
@@ -1,24 +1,50 @@
+import { unstable_cache } from "next/cache";
 import { cookies, headers } from "next/headers";
 import { redirect } from "next/navigation";
 import React from "react";
 
 import { getServerSession } from "@calcom/features/auth/lib/getServerSession";
+import type { AppFlags } from "@calcom/features/flags/config";
+import { FeaturesRepository } from "@calcom/features/flags/features.repository";
 
 import { buildLegacyRequest } from "@lib/buildLegacyCtx";
 
 import type { SettingsLayoutProps } from "./SettingsLayoutAppDirClient";
 import SettingsLayoutAppDirClient from "./SettingsLayoutAppDirClient";
 
+const getTeamFeatures = unstable_cache(
```

Comment:
When does it get invalidated? Maybe I am not finding it. If so, can you point me to the file with revalidation logic?

Reply:
It doesn’t - team features are only added in db rn

Reply:
So, team features are manually added to DB by us? And that manual addition will revalidate this cache, you mean?